### PR TITLE
About OpenRCT2 window tab, convert changelog to textreader

### PR DESCRIPTION
**Pull request by wolfreak99**
This is an experimental idea. A small complaint i've had in the back of my head was the lack of ability to open the changelog inside the game and the fact that "About OpenRCT2" opens up the original games about window, with no dedicated about for OpenRCT2. This introduces a tabbed About window with snippets from the OpenRCT2 license header. It also includes a textreader window that replaces the changelog window, that way both changelog.txt and contributors.md can be viewed within game, it also includes a link to view the github page in your browser.

This is likely not needed before the next stable release as it has a bit of code added outside of the About window, but I am not one to judge all on my own without feedback from others.

### CODE REVIEW

File: data/language/en-GB.txt [hunk 1]
```diff
@@ -3682,7 +3682,7 @@ STR_5340    :{SMALLFONT}{BLACK}Clearance height
 STR_5341    :Flags
 STR_5342    :Choose a map tile
 STR_5343    :Automatically place staff
-STR_5344    :Changelog
+STR_5344    :<not used anymore>
```
Comment (duncanspumpkin) [inline data/language/en-GB.txt]: Why are you removing this one and adding a new entry below. All your doing is throwing out work that the translators do.

**Comment by wolfreak99** [comment]
I didn't think of that, I think I did it to keep locations consistant for button texts but that definitely nullifies my reason.. I'll fix it as soon as I get home. Thanks duncanspumpkin

File: data/language/en-GB.txt [hunk 2]
```diff
@@ -4228,6 +4228,13 @@ STR_5916    :{COMMA16} player
 STR_5917    :{COMMA16} players
 STR_5918    :{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{POP16}{COMMA16}
 STR_5919    :{COMMA16}
+STR_5920    :Contributors
+STR_5921    :OpenRCT2 @ Github
+STR_5922    :Text Reader ({STRING})
+STR_5923    :Text Reader
+STR_5924    :OpenRCT2, a free and open source clone of Roller Coaster Tycoon 2.
+STR_5925    :OpenRCT2 is the work of many authors, a full list can be found in "Contributors". For more information, visit http://www.github.com/OpenRCT2/OpenRCT2
+STR_5926    :OpenRCT2 is not affiliated with Twitch or Github, all 3rd party copyrights belong to their respectable owners
```
Comment (Gymnasiast) [inline data/language/en-GB.txt]: `respective owners`

**Comment by Nubbie** [comment]
Fixes #3941

**Comment by mrtnptrs** [comment]
Looks great and works fine to me! :)

**Comment by janisozaur** [comment]
If only it compiled…

File: src/windows/textreader.c [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+static void window_textreader_paint(rct_window *w, rct_drawpixelinfo *dpi)
+{
+	window_draw_widgets(w, dpi);
+}
+
+static void window_textreader_scrollpaint(rct_window *w, rct_drawpixelinfo *dpi, int scrollIndex)
+{
+	gCurrentFontFlags = 0;
+	gCurrentFontSpriteBase = FONT_SPRITE_BASE_MEDIUM;
+	gfx_draw_string(dpi, RCT2_ADDRESS(0x009C383D, char), 1, dpi->x, dpi->y);
```
Comment (IntelOrca) [inline src/windows/textreader.c]: You should no longer use RCT2_GLOBAL or RCT2_ADDRESS.

**Comment by wolfreak99** [comment]
@janisozaur  Okay everything is compilable on both appveyor and travis-ci, with the only exception being the xcode files. not sure how to update that portion.

**Comment by wolfreak99** [comment]
Because my commits were a bit out of whack with the data and string_ids being updated alot throughout the commits, to make it easier on rebasing this, i squashed the commits down to just one, that way I only have to correct merge conflicts just once. Otherwise, I have to fix the merge conflicts like 5 seperate times.

**Comment by janisozaur** [comment]
@wolfreak99 are you going to revive this PR?

**Comment by wolfreak99** [comment]
@janisozaur I just seen the comment you left, apologies about the late delay. I really don't know what Is going on with me not getting notifications on here.. anyway, I'll rebase this as soon as I get home.. I will also try to reorganise the commits to be more clearer with what is what.

**Comment by Gymnasiast** [comment]
As with the other textreader PR, we agree with the principle but we want to do a different approach. I'm going to mark this as information for reference.

**Pull request closed without merging**